### DISCUSSION TITLE
My student benefits request was accepted but Im not getting copilot or any other benefit

### DISCUSSION THREAD

**Discussion by vspimentel** [community/community#124185] [Copilot Conversations]
### Select Topic Area

General

### Body

My student benefits request was accepted but Im not getting copilot or any other benefit 
![Screenshot 2024-05-16 212400](https://github.com/community/community/assets/81595020/ba099fd8-e668-46cb-8425-8f972230e998)

![Screenshot 2024-05-16 212507](https://github.com/community/community/assets/81595020/c4f51fe3-04e2-4fb6-8aac-9b1a38160661)

I know it says that they will come soon but its already two days, how much I have to wait?

**Comment by Minurte1** [comment]
Response #1

**Comment by davevad93** [comment]
Hi @vspimentel  ,

You have to wait for you benefits to be available, the message states that your benefits will be available soon, when your benefits will be available it shows the following message:

`Congrats! Your request for @vspimentel was approved and your benefits are now available.`

Generally you have to wait 2-3 days but the waiting time may vary depending on different factors, there are people who have had to wait 9-10 days or more.
